src/smt/smt_engine.cpp

```diff
@@ -4862,33 +4873,39 @@ void SmtEngine::checkSynthSolution()
     Node conj = Node::fromExpr(*i);
     // Apply any define-funs from the problem.
     {
-      unordered_map<Node, Node, NodeHashFunction> cache;
+      std::unordered_map<Node, Node, NodeHashFunction> cache;
       conj = d_private->expandDefinitions(conj, cache);
     }
     Notice() << "SmtEngine::checkSynthSolution(): -- expands to " << conj << endl;
     Trace("check-synth-sol") << "Expanded assertion " << conj << "\n";
-    if (conj.getKind() != kind::FORALL)
+    if (conjs.find(conj) == conjs.end())
```

Comment:
In retrospect `conj` was a poor choice for naming the local variable with the assertion. Could you rename it for "assertion" or something?

Reply:
Sure.